source/screens/caseScreens/CaseSummary.js

```diff
@@ -599,6 +690,20 @@ const CaseSummary = (props) => {
                 </Card>
               );
             })}
+
+            {journals.journal && journals.journal.notes.note.length > 0 && (
+              <Card>
+                <Card.Body color="neutral">
+                  <Card.Title colorSchema="neutral">Anteckningar</Card.Title>
+                  {journals.journal.notes.note.map((note) => (
+                    <>
+                      <Card.Text strong>{note.label}</Card.Text>
```

Comment:
Should probably replace `strong` with `type="h3"` to closer match the design (this simply adds a bit of margin but it looks nicer).